x/staking/src/types/keys.rs

```diff
@@ -0,0 +1,251 @@
+use gears::{
+    tendermint::types::time::timestamp::Timestamp,
+    types::address::{AccAddress, ValAddress},
+};
+
+use crate::consts::keeper::{
+    REDELEGATION_BY_VAL_DST_INDEX_KEY, REDELEGATION_BY_VAL_SRC_INDEX_KEY, REDELEGATION_KEY,
+    UNBONDING_DELEGATION_BY_VAL_INDEX_KEY, UNBONDING_DELEGATION_KEY, UNBONDING_QUEUE_KEY,
+};
+
+/// Returns a key prefix for indexing a redelegation from a delegator
+/// and source validator to a destination validator.
+pub fn redelegation_key(
```

Comment:
Should it be fn for Redelegation struct?

Reply:
I think it's fine with al the other keys.

Reply:
Okay